packages/gateway/package.json

```diff
@@ -52,5 +52,13 @@
   ],
   "main": "./build/js/index.js",
   "module": "./build/js/index.mjs",
-  "types": "./build/types/index.d.ts"
+  "types": "./build/types/index.d.ts",
+  "sls": {
+    "dependencies": {
+      "com.palantir.foundry.api:api-gateway": {
+        "minVersion": "1.763.0",
+        "maxVersion": "1.x.x"
+      }
```

Comment:
just so i understand, is this so this will propagate to the asset that the generator is included in? apologies i hadn't been following ultra closely

Reply:
This is halfway to that. Right now we're just adding it to @osdk/gateway so that anyone that depends on that will get it in their package tree so our internal code should pick this up. It doesn't wind up on the package built by foundry-sdk-generator OR the create-osdk-app assets.

Reply:
ok great, thanks for explaining! that's what I was hoping for